cmd/lakectl/cmd/fs_presign.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/go-openapi/swag"
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/api/apigen"
+)
+
+var fsPresignCmd = &cobra.Command{
+	Use:               "presign <path URI>",
+	Short:             "return a pre-signed URL for the specified object",
```

Comment:
Do you think we should mention it's a url for reading only?

Reply:
Done